mirrulations-client/src/mirrclient/client.py

```diff
@@ -298,7 +298,8 @@ def send_job(self, job_id, job_result, job_type):
             'job_id': job_id,
             'results': job_result
         }
-        if 'errors' not in job_result:
+        # If the job is not an attachment job we need to add an output path
```

Comment:
We should only need to send an output path when it is a "regular" job. If it's an attachment, the work server will just save the results (attachment files) in a constant location